### ISSUE TITLE
Accessibility Labels missing in Blocks within Column Blocks

### ISSUE THREAD

**Issue by zachspears** [wordpress-mobile/WordPress-iOS#16578]
*Please Note: To test this issue I turned on [Voice Control](https://support.apple.com/en-us/HT210417), I found this exceptionally useful for visualizing the accessibility support in the app.*

### Expected behavior
When using the app with accessibility functionality, the user should be able to read the Accessibility Labels, especially while adding text. A good example of this is while adding paragraph text, if you are using voice control you can see that accessibility is able to "see" that label and provide it to the user. 

See: 29786725-hc

### Actual behavior
When a block is inside of a Column Block, not all accessibility labels are available. For example when typing out a paragraph, similar to the above example, the accessibility label will not be available. I have also noticed the accessibility labels for the individual blocks and row buttons do not show up either.

### Steps to reproduce the behavior
1. Turn on [Voice Control](https://support.apple.com/en-us/HT210417).
2. Open any site Page or Post.
3. Add a column block.
4. Add any block with text options.
5. Type in the text field. Note it does not display an Accessibility Label.

##### Tested on iPhone 12 Mini, iOS 14.5, WPiOS 17.3

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
